pages/index.js

```diff
@@ -8,6 +9,7 @@ export default function Home() {
   return (
     <>
      <Wrapper>
```

Comment:
I think, for better practice, you should apply the wrapper to the section instead of applying it to the index .